Specs/Scene/ImplicitTilesetSpec.js

```diff
@@ -38,13 +38,13 @@ describe("Scene/ImplicitTileset", function () {
   var baseResource = new Resource("https://example.com/tileset.json");
   var contentUriTemplate = new Resource(contentUriPattern);
   var subtreeUriTemplate = new Resource(subtreeUriPattern);
-  var mockTileset = {};
+  var metadataSchema; // intentionally left undefined
 
   it("gathers information from both tile JSON and extension", function () {
     var implicitTileset = new ImplicitTileset(
-      mockTileset,
       baseResource,
-      implicitTileJson
+      implicitTileJson,
+      metadataSchema
```

Comment:
Also, it would be good to add a test that this metadata is accessible if this does get passed in. I've been putting things in different describe blocks by extension, so that would probably go best in something like:

```
describe("3DTILES_metadata", function() {
  it("stores metadataSchema", function() {
     // construct an implicit tileset and check for metadataSchema
  });
}):
```